src/main/resources/application.properties

```diff
@@ -23,3 +23,8 @@ management.metrics.web.server.request.autotime.enabled=false
 management.metrics.enable.logback=false
 management.metrics.enable.system=false
 management.metrics.enable.spring.integration=false
+
+# caching (override type in application.properties to enable)
+spring.cache.type=none
+spring.cache.cache-names=form_session
+spring.cache.caffeine.spec=maximumSize=500,expireAfterAccess=300s
```

Comment:
Do we know what to anticipate WRT the memory impact of caching these objects? seems easy for this to take ~1gb of heap or so. That feels very worth the hit to me, but wasn't sure if you'd thought it through in more detail.

Reply:
We could adjust this strategy to be based on the actual memory size but that requires some custom config. I figured based on the size of the objects in the DB that 500 would take up about 1GB of memory (on average). This can also be adjusted via local config if necessary.